pkg/instrumentation/sdk.go

```diff
@@ -291,12 +294,26 @@ func (i *sdkInjector) addParentResourceLabels(ctx context.Context, uid bool, ns
 			}
 			// parent of ReplicaSet is e.g. Deployment which we are interested to know
 			rs := appsv1.ReplicaSet{}
-			// ignore the error. The object might not exist, the error is not important, getting labels is just the best effort
-			//nolint:errcheck
-			i.client.Get(ctx, types.NamespacedName{
-				Namespace: ns.Name,
-				Name:      owner.Name,
-			}, &rs)
+			nsn := types.NamespacedName{Namespace: ns.Name, Name: owner.Name}
+			backOff := wait.Backoff{Duration: 10 * time.Millisecond, Factor: 1.5, Jitter: 0.1, Steps: 20, Cap: 30 * time.Second} // TODO decide which of these we need and what they should be set to
```

Comment:
What is the pod webhook timeout? cap of 30 seconds seems too much TBH.

Could we extract the retry functionality to a separate function? It will be useful if we want to reuse it for other objects.

Reply:
The pod webhook timeout is 10 seconds according to https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/

Reply:
max wait 10 seconds is too much. Could we use e.g. 2-3 seconds?